src/main/resources/templates/templateOpportunityNotification.xml

```diff
@@ -52,10 +54,14 @@
                 <div>
                     <span class="tempo-text-color--secondary">Close Date:</span>
                     <span>${entity['OpportunityNotification'].current.opportunity.CloseDate ! "-"}</span>
-                    <span class="tempo-text-color--secondary">Probability:</span>
+                    <span class="tempo-text-color--secondary">&#160;&#160;&#160;Probability:</span>
```

Comment:
If use various spaces, the browser render one unique space, So we use the ASCII code.